src/js/themes/base.js

```diff
@@ -1274,9 +1274,21 @@ export const generate = (baseSpacing = 24, scale = 6) => {
           opacity: 'weak',
         },
         color: 'status-critical',
+        // message: {
```

Comment:
do we also need 
```
        // message: {
        //   color: undefined,
        // },
        // title: {
        //   color: undefined,
        // },
```
above under just `notification.title` ? I see you have it under all the `critical` `warning` `normal` etc but I think we need it above as well?

Reply:
do we also need under `notification.global`?

Reply:
The naming structure is intended to be `theme.notification.[status].[kind].message` -- so it's not needed under `notification.global`

Reply:
aw yes you are right thought I saw this in the `base.d.ts` file but thanks for clarifying